### ISSUE TITLE
Front page app invisible text in dark mode

### ISSUE THREAD

**Issue by ahuang11** [holoviz/panel#6894]
The widgets labels are invsible.

<img width="1334" alt="image" src="https://github.com/holoviz/panel/assets/15331990/09cb455d-05be-43b1-9619-dce38fc38109">

**Comment by philippjfr** [comment]
Thanks, until we have CSS styling of the plot we should disable dark mode again.

**Comment by Coderambling** [comment]
Disable it only on that and similar pages I assume? Not actually disable it for the whole site?

**Comment by philippjfr** [comment]
> Disable it only on that and similar pages I assume? Not actually disable it for the whole site?

Yes for the whole site, a) there's no way to do it for specific pages and b) we will never catch all the pages where there might be an issue.

**Comment by Coderambling** [comment]
Oof that seems quite a downgrade in terms of site functionality.

And it also doesn't look good in terms of perception.

I assume the invisible text issue occurs multiple times?

Users can probably fall back to a browser-level Dark Mode toggle, but still.

Is there a development path / PR 
to enable CSS for plots, so that there is a way back to re-enable dark mode at some point?

**Comment by philippjfr** [comment]
>Oof that seems quite a downgrade in terms of site functionality.

We had dark mode disabled for the longest time for this reason but recent versions of the pydata-sphinx-theme changed the APIs we used to disable it.

>to enable CSS for plots, so that there is a way back to re-enable dark mode at some point?

Yes, it's being worked on in Bokeh 3.5 and 3.6.
